### ISSUE TITLE
Layout: Improve support for multi-line axis labels

### ISSUE THREAD

**Issue by CBrauer** [ScottPlot/ScottPlot#4535]
**Suggestion:** 
A multi-line title works fine.  But a multi-line X-axis label does not work.

Here is what I have tried:
```cs
ScottPlot.Plot plt = new();
plt.ScaleFactor = 2;
plt.Clear();

double[] xs = new double[] { 1, 2, 3, 4, 5, 6, 7 };
double[] ys = new double[] { 2, 2, 3, 3, 3.8, 4.2, 4 };

// plot original data as a scatter plot
var scatter = plt.Add.Scatter(xs, ys);
scatter.LineWidth = 0;
scatter.MarkerSize = 10;
scatter.Axes.YAxis = plt.Axes.Right;

// calculate the regression line
ScottPlot.Statistics.LinearRegression reg = new(xs, ys);

// plot the regression line
Coordinates pt1 = new(xs.First(), reg.GetValue(xs.First()));
Coordinates pt2 = new(xs.Last(), reg.GetValue(xs.Last()));
var line = plt.Add.Line(pt1, pt2);
line.Color = Colors.Orange;
line.MarkerSize = 0;
line.LineWidth = 4;
line.LinePattern = LinePattern.Dashed;
line.Axes.YAxis = plt.Axes.Right;

var titleString = $"This is a three line title.\nAnd here is line two.\nAnd here is line three.";
plt.Title(titleString);
plt.Axes.Title.Label.FontName = "Verdana";
plt.Axes.Title.Label.FontSize = 12;
plt.Axes.Title.Label.Bold = false;

var descriptionString = $"But sadly a three line description does not work.\nAnd here is line 2.\nAnd here is line 3.";
plt.XLabel(descriptionString);
plt.Axes.Bottom.Label.FontName = "Verdana";
plt.Axes.Bottom.Label.FontSize = 12;
plt.Axes.Bottom.Label.Bold = false;

// Render the plot
plt.SavePng("../../../scatter_with_limit.png", 800, 800);
```

With the  following result:
![scatter_with_limit](https://github.com/user-attachments/assets/6c709677-7804-46e2-abf5-7cb7115699c6)

Charles

**Comment by swharden** [comment]
Simplified reproduction of this issue: 

```cs
public partial class Form1 : Form
{
    public Form1()
    {
        InitializeComponent();

        formsPlot1.Plot.Add.Signal(Generate.Sin());
        formsPlot1.Plot.Add.Signal(Generate.Cos());

        formsPlot1.Plot.XLabel("Line 1\nLine 2\nLine 3");
    }
}
```

![image](https://github.com/user-attachments/assets/85bae534-6344-4568-b2a5-1143020cacf9)

**Comment by swharden** [comment]
Fixed in #4545

before | after
---|---
![image](https://github.com/user-attachments/assets/1ea1c7af-1d42-469c-a0c5-de82d89402f3)|![image](https://github.com/user-attachments/assets/cb6fc5ff-552a-4ebf-984e-01c283927db9)
